ocaml/testsuite/tests/typing-unique/unique.ml

```diff
@@ -542,38 +542,38 @@ type box = { x : int }
 type box = { x : int; }
 |}]
 
-let curry (unique_ b1 : box) (unique_ b2 : box) = b1
+let curry (unique_ b1 : box) (unique_ b2 : box) = ()
```

Comment:
Why the changes here? These seem to test a little less than before.

Reply:
I suppose the thing we are testing with this series of `curry` is just the logic around uncurried functions. If we say `b1` or `b2`, it would also test the logic around `Env.lock_mode` and introduce noise. The latter is covered by other tests.